components/ThePlayground.client.vue

```diff
@@ -14,6 +13,12 @@ const isDragging = usePanelDragging()
 const panelSizeEditor = usePanelCookie('nuxt-playground-panel-editor', 30)
 const panelSizeFrame = usePanelCookie('nuxt-playground-panel-frame', 30)
 
+const { iframeLocation, wcUrl } = usePlayground()
+
+// auto update inputUrl when location value changed
+const inputUrl = ref<string>('')
+syncRef(inputUrl, computed(() => iframeLocation.value.fullPath), { direction: 'rtl' })
```

Comment:
Not an logical issue, but maybe `ltr` would follows the intuition more?

Reply:
um.. you mean like this?
```suggestion
syncRef(computed(() => iframeLocation.value.fullPath, inputUrl), { direction: 'ltr' })
```

Reply:
Yes, thanks

Reply:
By the way, feel free to check my recent PR 😂 https://github.com/vueuse/vueuse/pull/3549
This can solve the issue with 1 line of code.